assets/app/data/crowbar/services/crowbar.factory.js

```diff
@@ -41,7 +41,7 @@
 
```

Comment:
Ahhh I see now! Indeed you are rigth, this should be moved into the appropiate place now that has changed places in the API